src/Core/src/VisualDiagnostics/DebuggerHelper.cs

```diff
@@ -1,17 +1,16 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
-using System;
 using System.Diagnostics;
 
-namespace Microsoft.Maui.Controls.Xaml.Diagnostics
+namespace Microsoft.Maui
 {
-	static class DebuggerHelper
+	public static class DebuggerHelper
```

Comment:
Can we keep this internal? At the very least, this must be in a separate namespace that no-one uses it. This is purely for our testing.

Reply:
I'm happy to leave it internal, I just couldn't figure out how to make it work, since I think that would mean giving Internal account to Maui.Controls for Maui.Core, and I'm not sure if that's wanted?

If there's a way to do it and keep it working, I'm 100% happy to do it.

Reply:
I think it is fine for now - especially since it is just for testing? The Controls library itself doesn't actually _use_ this. Or it shouldn't.

Reply:
Ah, I'm dumb. I didn't notice that Internals were already exposed for Maui.Controls. I needed to add Controls.Xaml to InternalsVisibleTo and now it all works.

https://github.com/dotnet/maui/pull/4333/commits/0310b4ab20916542bb11a521a3893571320d61cb